src/main/java/it/pagopa/ecommerce/payment/methods/application/v2/PaymentMethodService.java

```diff
@@ -154,6 +154,25 @@ private CalculateFeeResponseDto bundleOptionToResponse(
                                 .taxPayerFee(t.getTaxPayerFee())
                                 .touchpoint(t.getTouchpoint())
                                 .pspBusinessName(t.getPspBusinessName())
+                ).sorted(
+                        (
+                         bundle1,
+                         bundle2
+                        ) -> {
+                            if (Boolean.TRUE.equals(bundle1.getOnUs())) {
+                                if (Boolean.TRUE.equals(bundle2.getOnUs())) {
```

Comment:
this case should never happen, that is, only one bundle can be on us (bin to psp is a 1:1 relationship)

Reply:
Handled in 667ce51c94970b6b067742d9c4133e9181ce1f2b